src/icons/File.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+export const WhpptIconFile: FC = ({}) => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="currentColor">
```

Comment:
This can be moved to assets as a svg file and imported directly. SVGR makes this possible